.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "sdk-core"]
 	path = packages/core-bridge/sdk-core
-	url = git@github.com:temporalio/sdk-core.git
+	url = ../sdk-core
```

Comment:
I think this should be HTTP. I changed to HTTP in SDK Python so people had better experience.